tools/render-test/options.cpp

```diff
@@ -54,32 +60,32 @@ SlangResult parseOptions(int* argc, char** argv)
         }
         else if( strcmp(arg, "-hlsl") == 0 )
         {
-            gOptions.rendererID = RendererID::D3D11;
```

Comment:
Nice. Thanks for doing this cleanup. It was annoying that the existing command line arguments were specifying both source language and renderer, but I was wary about changing the behavior since a bunch of tests assumed the existing meaning. This is a nice fix that retains compatibility with the old tests.